docs/introduction/entities.md

```diff
@@ -249,6 +356,16 @@ class BrandData
 }
 ```
 
+### Data for multidomain or multilanguage field
+
+As you can see in example above, multidomain or multilanguage field is an array.
+
+Multidomain field like `seoH1s` have to be indexed by `domainId` - integer ID of the given domain.
+Data factory have to prepare this array to contain keys for all domain IDs, because domain entities are created by these array items (even if their value is null).
+
+Multilanguage field like `description` have to indexed by `locale` - string identifier of language (you can find them in [`domains.yml`](/project-base/app/config/domains.yml)).
+Data factory do not prepare this arrays, because the extension we use for translated entities can handle unprepared translations.
```

Comment:
```suggestion
Data factory does not need to prepare this array, because the extension we use for translated entities can handle unprepared translations.
```